OpenTrons/opentrons-api
# fix(components, protocol-designer): fix well order and tip position advanced settings

# Overview

Fix styling for advanced settings PositionField and WellsOrderFields components. Adds icons for each ListButton appropriately and reformats text. Adds new icon for well-order

Closes RQA-3499

## Test Plan and Hands on Testing

- navigate to transfer advanced settings
- verify that well order and tip position icons show properly, and that tip position text is formatted correctly

## Changelog

- update `WellsOrderField` and `TipPosition` `ListButton` children
- add icon
- fix translation

## Review requests

- see test plan

## Risk assessment

low
